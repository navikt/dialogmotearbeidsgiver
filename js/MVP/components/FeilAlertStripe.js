import React from 'react';
import styled from 'styled-components';
import AlertStripe from 'nav-frontend-alertstriper';

const AlertStripeStyled = styled(AlertStripe)`
  margin-bottom: 32px;
`;

const texts = {
  feil:
    'Akkurat nå mangler det noe her. Vi har tekniske problemer som vi jobber med å løse. Prøv gjerne igjen om en stund.',
};

const FeilAlertStripe = () => {
  return <AlertStripeStyled type="feil">{texts.feil}</AlertStripeStyled>;
};

FeilAlertStripe.propTypes = {};

export default FeilAlertStripe;
