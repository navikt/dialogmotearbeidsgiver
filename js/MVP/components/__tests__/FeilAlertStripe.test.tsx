import { render } from '@testing-library/react';
import React from 'react';
import FeilAlertStripe from '../FeilAlertStripe';

describe('FeilAlertStripe', () => {
  test('should render', async () => {
    const { container } = render(<FeilAlertStripe />);

    expect(container).toHaveTextContent('Akkurat nÃ¥ mangler det noe her.');
  });
});
