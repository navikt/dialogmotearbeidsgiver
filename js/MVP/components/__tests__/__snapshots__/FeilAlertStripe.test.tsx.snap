// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeilAlertStripe should render 1`] = `
<div>
  <div
    class="alertstripe sc-bdvvtL HFdGN alertstripe--feil"
  >
    <span
      class="alertstripe__ikon"
    >
      <span
        class="sr-only"
      >
        feil
      </span>
      <svg
        aria-label="feil-ikon"
        focusable="false"
        height="1.5em"
        kind="feil-sirkel-fyll"
        role="img"
        viewBox="0 0 24 24"
        width="1.5em"
      >
        <g
          fill="none"
          fill-rule="evenodd"
        >
          <path
            d="M11.999 0C5.395 0 .013 5.372 0 11.976a11.923 11.923 0 0 0 3.498 8.493A11.925 11.925 0 0 0 11.977 24H12c6.603 0 11.986-5.373 12-11.978C24.013 5.406 18.64.012 11.999 0z"
            fill="#A13A28"
            fill-rule="nonzero"
          />
          <path
            d="M12 10.651l3.372-3.372a.954.954 0 1 1 1.349 1.35L13.349 12l3.372 3.372a.954.954 0 1 1-1.35 1.349L12 13.349 8.628 16.72a.954.954 0 1 1-1.349-1.35L10.651 12 7.28 8.628A.954.954 0 1 1 8.63 7.28L12 10.651z"
            fill="#FFF"
            fill-rule="nonzero"
          />
        </g>
      </svg>
    </span>
    <div
      class="typo-normal alertstripe__tekst"
    >
      Akkurat nå mangler det noe her. Vi har tekniske problemer som vi jobber med å løse. Prøv gjerne igjen om en stund.
    </div>
  </div>
</div>
`;
